### PR TITLE
chore(airflow): Copernicus API weather collection DAG for InfoDengue (Catchup)

### PR DESCRIPTION
**Pull request by luabida**
This DAG fetches data from Copernicus API monthly since 2000-01-01 towards.

Needs extra configuration because it will store the data at InfoDengue SQL server, I've been using a `.env` file at the same directory of the DAG, any thoughts about this approach? @xmnlab

### CODE REVIEW

**Comment by luabida** [comment]
@xmnlab > So it should be in the main .env 

After this, in order to pass on CI, I would have to expose AD's postgres config, is there any better solution to this?

**Comment by xmnlab** [comment]
sorry .. not sure if I am following you here, why do you need to expose that?

**Comment by luabida** [comment]
@xmnlab sorry for the delay, I had a exhausting docker package error (not sure if k8s deconfigured it somehow)

I'm referring about the github CI ENV variables for the workflow, I'll send you mine .env to exemplify

File: containers/airflow/dags/copernicus_api/brasil.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                "umid_med",
+                "umid_max",
+            ]
+        )
+
+        total_cities = 5570
+        with tqdm(total=total_cities) as pbar:
+            for geocode in geocodes:
+                row = extract_reanalysis.netcdf_to_dataframe(file, geocode)
+                cope_df = cope_df.merge(row, on=list(cope_df.columns), how="outer")
```
Comment (luabida) [inline containers/airflow/dags/copernicus_api/brasil.py:129]: This line takes a long time, does someone know a better approach to this?

**Pull request closed without merging**